### ISSUE TITLE
`utils.expand_pretext_href`

### ISSUE THREAD

**Issue by StevenClontz** [PreTeXtBook/pretext-cli#131]
I think I figured out the "right" way to write XSL that wants to reference PreTeXt XSL.

Rather than write

```
<xsl:import href="pretext-common.xsl"/>
```

the author could write

```
<xsl:import pretext-href="pretext-common.xsl"/>
```

Then when the PreTeXt CLI reads this as XML, it knows where the PreTeXt XSL we distribute lives, so we can use LXML to replace all instances of `@pretext-href` like this:

```
<xsl:import href="/path/to/package/pretext/static/xsl/pretext-common.xsl"/>
```

**Comment by StevenClontz** [comment]
This is basically what I do manually here to get a copy of `pretext-common.xsl` into a custom XSL for turning our book into slides: https://github.com/StevenClontz/cal2-tbil-2021/blob/c0ffa50beacf198c1e42963243f23fe1db020291/build_html_with_slides.py#L18

**Comment by StevenClontz** [comment]
Couldn't help myself, closed by #142
